engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActorSpec.scala

```diff
@@ -178,7 +179,7 @@ class EngineJobExecutionActorSpec extends CromwellTestkitSpec with Matchers with
 
       def ejeaInRunningState() = {
         val ejea = buildEJEA(restarting = true)
-        ejea.setState(stateName = RunningJob, stateData = EmptyPartialCompletionData)
+        ejea.setState(stateName = RunningJob, stateData = NoResponseData(null, Props.empty))
```

Comment:
we have a whole bunch of new states and execution pathways. We should check that the outcome is what we expect given a variety of situations